burst/src/Controllers/BurstMetricsController.cs

```diff
@@ -39,14 +39,15 @@ public IActionResult Get(string ns, string deployment)
             // But we can control what to output if we do null
             // TODO: Set the default value from appsettings.json
 
-            string cpuTarget = hpaMetrics?.TargetCPULoad?.ToString() ?? "-1";
-            string cpuCurrent = hpaMetrics?.CurrentCPULoad?.ToString() ?? "-1";
+            string targetLoad = hpaMetrics?.TargetLoad?.ToString() ?? "-1";
+            string currentLoad = hpaMetrics?.CurrentLoad?.ToString() ?? "-1";
+            string maxLoad = hpaMetrics?.MaxLoad?.ToString() ?? "-1";
 
```

Comment:
In what way are the -1 values intended to be used?  I see no conditional for this value so currently it looks like if no values come back then this method would return a header with all -1 values.  I don't think that is what we want.  I'm okay with the nullables, because it means we don't have to build some complex error handling, but maybe instead of using a string have it return the value and then test for the -1 in any of the values and if any of them are -1 then return a 204 with an empty body.

Reply:
I agree, as part of https://github.com/retaildevcrews/wcnp/issues/294, check if any value is not available (i.e. null), then return 204 without any content.

We could check if the metrics are null (since they are nullable)

```C#
# pseudo code
if (hpaMetrics?.TargetLoad == null ||
    hpaMetrics?.CurrentLoad == null || 
    hpaMetrics?.MaxLoad == null)
{
    return <HTTP 204 status with empty body>;
}
else
{
    <previous code block with 200-content goes here>
}
```

Thoughts?

Reply:
Whether it's null or -1 is a decision that needs to be made then what we do with it is the next part. We can take care of what to do with it in the next PR, but I'm this PR you're assigning the value so we should decide which value to use and why.